odoo/addons/base/views/res_users_identitycheck_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="identity_check_wizard" model="ir.ui.view">
+            <field name="model">res.users.identitycheck</field>
+            <field name="arch" type="xml">
+                <form string="Password Confirmation">
+                    <sheet class="bg-primary">
+                        <h3><strong>Please enter your password to confirm you own this account</strong></h3>
+                        <br/>
+                        <div>
+                            <field class="o_field_highlight col-10 col-md-6 px-0" name="password" autocomplete="current-password"
+                                required="True" password="True" placeholder="************"/>
+                        </div>
+                        <a href="/web/reset_password/" class="btn btn-link" role="button">Forgot password?</a>
+                    </sheet>
+                    <footer>
+                        <button string="Confirm Password" type="object" name="run_check" class="btn btn-primary" data-hotkey="q"/>
+                        <button string="Cancel" special="cancel" data-hotkey="z" class="btn btn-secondary"/>
+                    </footer>
+                </form>
+            </field>
+        </record>
+
+        <record id="identity_check_revoke_all_devices_view_form" model="ir.ui.view">
```

Comment:
Same here, to update

Reply:
As the id of the view just above is "identity_check_wizard" and shouldn't change because of a required migration process, I would think it could be better to keep the id's uniform inside the file even if it makes them a little less uniform with the rest of Odoo and therefore keep this id written like this too. This one doesn't need a migration process as it is new and therefore, if you tell me that it would be better to uniform it to Odoo rather than the id straight above, I'll rename it. Is it ?

Reply:
Well as we target master, we may make a small imp commit indeed :) as anyway we are passing through, let us clean it quickly.

Personally I would rename all once and make them guideline compliant, as the opportunity is there as we are modifying the file. But that's not mandatory, just something I would do.